### PR TITLE
Update notifications page

### PR DESCRIPTION
**Pull request by bruce-thomas**

### CODE REVIEW

File: src/content/1.7/themes/reference/templates/notifications.md [hunk 1]
```diff
@@ -8,12 +8,12 @@ aliases:
 # Notifications
 
 Throughout the whole front office, the customer can receive notification messages
-from PrestaShop, to inform her about successes or errors for instance.
```
Comment (sarjon) [inline src/content/1.7/themes/reference/templates/notifications.md]: i see women getting mad for this. :trollface: can we rephrase it or keep `her / him` maybe?

Comment (Quetzacoalt91) [reply src/content/1.7/themes/reference/templates/notifications.md]: You can actually remove `her` without changing the meaning of the sentence. :)

Comment (bruce-thomas) [reply src/content/1.7/themes/reference/templates/notifications.md]: I have pushed the modification. The commit message was hard to find ;)

**Comment by mickaelandrieu** [comment]
Thank you for the contribution @bruce-thomas.

Also,

> Gender equality for customers

we will always merge updates that will improve gender neutrality and make our communication more welcoming for everyone!

Mickaël

**Pull request merged**